### ISSUE TITLE
"Accessibility Supported" might benefit from mentionning accessibility APIs

### ISSUE THREAD

**Issue by A11yTea** [w3c/wcag#4868]
Hi everyone,

I filed an issue for WCAG-EM that might impact WCAG, its why I want to also post it here. ["Accessibility Supported" and Accessibility APIs... #150](https://github.com/w3c/wai-wcag-em/issues/150). 

To summarize, I want to inquire if accessibility supported should make mention of accessibility APIs. Especially since if technologies build to the APIs rather than specific assistive technology, it allows for more support for all other assistive technologies.

I know 4.1.2 Name, Role, Value supports accessibility APIs but since they are central to interoperability, I want to know if content should be added to explain those APIs in the Accessibility Supported section.
